src/components/TodoModal/TodoModal.tsx

```diff
@@ -2,11 +1,26 @@
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+interface Props {
+  chosenTodo: Todo;
+  onChoosingTodo: (todo: Todo | null) => void;
```

Comment:
Consider following a naming convention for your event handler functions and props. A common practice is to use 'handle' for methods and 'on' for props, like 'handleChooseTodo' and 'onChooseTodo'. It will make your code more readable and easier to maintain. Here is a [reference](https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364) for more details.